### PR TITLE
Add an account-balance command for checking account balances

### PR DESCRIPTION
**Pull request by jwiegley**
Example, you can check the account balance at https://ic.rocks/account/ec0e2456fb9ff6c80f1d475b301d9b2ab873612f96e7fd74e7c0c0b2d58e6693 using:
```
quill account-balance ec0e2456fb9ff6c80f1d475b301d9b2ab873612f96e7fd74e7c0c0b2d58e6693
```

### CODE REVIEW

File: tests/outputs/account-balance.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Sending message with
```
Comment (chmllr) [inline tests/outputs/account-balance.txt:1]: This is the test output, but the new command file is missing?

**Review by chmllr — COMMENTED** [review]
Thanks a lot John! I think we miss the test file and I'm not sure why you mention ic.rocks. I initially thought this would _query_ ic.rocks page for the balance 😅 Maybe add that it's displaying the same info as on ic.rocks or https://dashboard.internetcomputer.org/account/ec0e2456fb9ff6c80f1d475b301d9b2ab873612f96e7fd74e7c0c0b2d58e6693

File: tests/outputs/account-balance.txt (hunk 1 above)
Comment (jwiegley) [reply tests/outputs/account-balance.txt:1]: oops, adding now

**Comment by jwiegley** [comment]
There is no connection to IC rocks, since that's not a truly anonymous way to query one's balance. I was just showing it as an example for verification of the balance figure from this command..

**Comment by jwiegley** [comment]
@chmllr Is it possible to implement this query without signing, so that no private key is needed to use the account balance query? Like for getting neuron info, it would be nice to be able to check balance and delay without needing a private key, which should only be necessary for checking maturity.

**Review by chmllr — APPROVED** [review]

**Pull request merged**